src/nls/urls.js

```diff
@@ -52,6 +52,7 @@ define(function (require, exports, module) {
         "sv": true,
         "zh-cn": true,
         "zh-tw": true,
-        "tr": true
+        "tr": true,
```

Comment:
Nit: these should be alphabetized, but the `tr` wasn't either so we can just fix that later